docs/tulokset.md

```diff
@@ -0,0 +1,3 @@
+# Mallin tulokset
+
```

Comment:
Mukaan myös mm. jako tuntiryhmäkohtaiset vs vrk-luvut + kulkutapaosuuksien laskenta

Reply:
@zptro

Reply:
Merviltä: Tärkeää käydä keskustelua siitä mille tuloksille on tarvetta ja tuoda näitä lisäksi aineistoon